apps/hmc/hmc-hmi-outbound-adapter/perftest.yaml

```diff
@@ -5,6 +5,17 @@ metadata:
 spec:
   values:
     java:
+      replicas: 2
+      autoscaling:
+        enabled: true
+        maxReplicas: 4
+      memoryRequests: '1536Mi'
+      memoryLimits: '2048Mi'
+      cpuRequests: '300m'
```

Comment:
I'm not sure cpuRequests needs to be increased from 250 to 300 based on current prod usage. However, it might be worth reverting cpuLimits back to 1500 so ensure the app isn't starved of CPU on startup.

Reply:
bump cpulinit and left request as is for now